compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmConstants.java

```diff
@@ -406,7 +408,10 @@ public class JvmConstants {
     public static final String OBSERVABLE_ANNOTATION = "ballerina/observe/Observable";
     public static final String DISPLAY_ANNOTATION = "display";
     public static final String RECORD_CHECKPOINT_METHOD = "recordCheckpoint";
-
+    public static final String CREATE_INTEROP_ERROR_METHOD = "createInteropError";
+    public static final String LAMBDA_PREFIX = "$lambda$";
+    public static final String POPULATE_METHOD_PREFIX = "$populate";
+    public static final String ADD_METHOD = "add";
```

Comment:
These are not specific to `// observability related constants` right? Shall we move these under `// code generation related constants`

Reply:
Other changes LGTM.